measurements-to-gelf.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+#
+# Prototype of a RIPE Atlas to GELF parser by awlx 
+#
+# Usage: python measurements-to-gelf.py <id> <min>
+#
+
+# Imports
+from gelfclient import UdpClient
+from urllib2 import urlopen
+import syslog
+import json
+import time
+import sys
+import requests
+
+# Configuration
+gelf_server = 'localhost'
+gelf_port = 5555
+api_key = '<api_key>'
+
+# Parse Args
+if len(sys.argv) == 1:
+    print "# Prototype of a RIPE Atlas to GELF parser"
+    print "#"
+    print "# Usage: python measurements-to-gelf.py <measurement id> <timeframe in min>"
+    print "#"
+    print "# Example: python measurements-to-gelf.py 12323 5"
+    print "#"
+    print "# Please define Server and Port inside the script and you need an API Key for Geolocation from https://geocoder.opencagedata.com/pricing"
+    exit(1) 
+
+measurement_id = str(sys.argv[1])
+timeframe_secs = int(sys.argv[2]) * 60
+
```

Comment:
I know it sounds silly, but the pep8 standard dictates two blank lines here and not three.